### PR TITLE
rename `--start-from-copy` to `--extend-run`

### PR DESCRIPTION
**Pull request by joaomcteixeira**
You are about to submit a new Pull Request. Before continuing make sure you read the [contributing guidelines](CONTRIBUTING.md) and you comply with the following criteria:

- [x] You have stick to Python. Talk with us before adding other programming languages to HADDOCK3
- [ ] Your PR is about CNS
- [x] Your code is well documented: proper docstrings and explanatory comments for those tricky parts
- [x] You structured the code into small functions as much as possible. You can use classes if there's a (state) purpose
- [x] code follows our coding style
- [x] You wrote tests for the new code
- [x] `tox` tests pass. *Run `tox` command inside the repository folder*
- [x] `-test.cfg` examples execute without errors. *Inside `examples/` run `python run_tests.py -b`*
- [x] PR does not add any *install dependencies* unless permission granted by the HADDOCK team
- [x] PR does not break licensing
- [ ] Your PR is about writing documentation for already existing code :fire:
- [ ] Your PR is about writing tests for already existing code :godmode:

---

While preparing the presentation slides for #425, I realized `--extend-run` would be a better name to `--start-from-copy` because of its mixed possibilities.

### CODE REVIEW

**Comment by rvhonorato** [comment]
Sorry I'm not in the loop about this feature, thanks for asking for my review tho! :)

File: docs/tutorials/continuing_runs.md [hunk 1]
```diff
@@ -61,6 +61,20 @@ We use `--restart 5` because the first new step is the sixth in the new
 workflow: the five initial steps and three new steps. Remember `--restart` is
 0-indexed.
 
+
+## Extend a run
+
+You can extend a successful run with additional steps. For that, prepare a
+configurtation file containing only the new steps you wish to execute on top of
+a previously successful run. In these cases, you **don't** need to define the
+`run_dir` and `molecules` parameters in this new configuration file. You **can**
```
Comment (mgiulini) [inline docs/tutorials/continuing_runs.md:70]: Just a curiosity: Does this mean that it is forbidden to define run_dir and molecules or just that they are not mandatory?

Comment (mgiulini) [reply docs/tutorials/continuing_runs.md:70]: In case they are forbidden I would substitute `don't need to` with `must not`

**Review by mgiulini — APPROVED** [review]

File: docs/tutorials/continuing_runs.md (hunk 1 above)
Comment (joaomcteixeira) [reply docs/tutorials/continuing_runs.md:70]: They will be ignored. Should I add that?

**Pull request merged**